### PR TITLE
clear labels for OSM hyperlinks in history panel (OSM history & OSM Profile)

### PR DESCRIPTION
**Pull request by jasonghent98**
Hi! I saw that there were no accepted pull requests to this issue, so I decided to take a stab at it to make it a little clearer in the history panel. I hope this works!

### CODE REVIEW

**Comment by tyrasd** [comment]
this is for issue #8784

**Review by tyrasd — CHANGES_REQUESTED** [review]
Can you please also change the label of the changeset link to osm as well (currently line 79)?

I'm wondering if we should make these texts translatable. I think it would not hurt. I can change that for you if you're not feeling comfortable, but it would need additional entries in [`data/core.yaml`](https://github.com/openstreetmap/iD/blob/develop/data/core.yaml) and the translated text needs to be appended here using `.call(t.append(…))` instead of `.text(…)`. See line 24 for an example.

//edit: there are also a few minor code style issues (a missing `;` and some superfluous space) to be fixed, see: https://github.com/openstreetmap/iD/pull/8834/files

**Comment by jasonghent98** [comment]
Hi,
Let me know if that looks better. In regards to making the texts translatable, are you referring to the core.yaml file itself, or rather the texts within the OSM history panel links (history, profile, changeset)? If the first, I could probably just use a third-party tool like iCanLocalize to do that. If the latter, I can also take care of that within the yaml file.

File: modules/ui/panels/history.js [hunk 1]
```diff
@@ -40,7 +40,7 @@ export function uiPanelHistory(context) {
                 .attr('class', 'user-osm-link')
                 .attr('href', osm.userURL(userName))
                 .attr('target', '_blank')
-                .text('OSM');
+                .text('OSM Profile');
```
Comment (1ec5) [inline modules/ui/panels/history.js]: Also call `t.append` here. The argument to `t.append` should refer to an entry in English that you’ll need to add to in data/core.yaml. Once this PR lands, translators will be able to provide translations using Transifex, so there’s no need to handle the translations yourself.

**Comment by tyrasd** [comment]
> In regards to making the texts translatable, are you referring to the core.yaml file itself, or rather the texts within the OSM history panel links (history, profile, changeset)?

I was referring to the texts of the links in the history panel. I've just written a small tutorial about how to do that here: https://github.com/openstreetmap/iD/wiki/How-to-use-iD%27s-localizer-module

**Comment by jasonghent98** [comment]
Thanks for the clarification. Let me know how this looks.

**Review by tyrasd — CHANGES_REQUESTED** [review]
Looks like it's almost done! Only two more things:

1. drop the `.text('…')` lines (lines 44, 81 and 204). The text adding is done already with the `.call(t.append(…));`
2. start all new strings in capital letters perhaps?

**Review by tyrasd — APPROVED** [review]

**Pull request merged**